src/app/app.component.scss

```diff
@@ -1,6 +1,19 @@
+@import '../abstract';
+
 #cookie-banner {
   position: fixed;
   bottom: 3em;
   right: 3em;
   z-index: 10;
 }
+
+.skip-to-content-link {
+  z-index: 10;
+  position: absolute;
+  transform: translateY(-100%);
+  transition: transform 0.3s;
+}
+
+.skip-to-content-link:focus-within {
```

Comment:
Does this need to be `:focus-within` because an inner shadow DOM containing the button might be what actually receives the focus? When I've implemented similar to this in the past I've pretty much followed [this](https://www.a11yproject.com/posts/how-to-hide-content/) and used simple `:focus`.

Reply:
I don't think it's an issue with the shadow dom, it's just because the div isn't focusable, its the button inside that is focusable.

Reply:
Maybe it is sort of a shadow dom thing, or just an encapsulation thing - if we wanted to apply a css rule on #focus we'd have to target the `a` element inside the button component. Focus-within seems to work well.

Reply:
Also could use a simple a tag instead of the button, but then it's hard to follow the consistent look and feel. I don't that's super important here though.